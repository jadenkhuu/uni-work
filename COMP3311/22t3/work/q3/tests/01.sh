psql racing -c "select * from q3('Guineas') order by race;"
