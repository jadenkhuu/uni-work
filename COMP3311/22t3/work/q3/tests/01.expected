         race         |         horses          
----------------------+-------------------------
 Australian Guineas   | F2,F3,F4,G2,M3,M4
 Caulfield Guineas    | F4,F5,G3,M3,M4
 Hawkesbury Guineas   | F5,G3,G5,M3,M3,M5
 Queensland Guineas   | F2,F3,F3,G3,G3,G3,M4
 Randwick Guineas     | F3,F3,F3,M4,M5
 Rosehill Guineas     | F4,F4,F5,G3,M3,M3,M4,M5
 The Thousand Guineas | F2,F4,F4,G3,G3,M4,S5
(7 rows)
